src/evict/evict_page.c

```diff
@@ -125,6 +125,7 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
      */
     time_start = 0;
     if (LF_ISSET(WT_EVICT_CALL_URGENT)) {
+        ++ref->page->page_eviction_force_count;
```

Comment:
Don't we just need a single bit? That is, if we're attempting to forcibly evict a page, we're never going to stop trying until we manage to evict it?

Seems like a flag would be sufficient.

Reply:
Good point